# RCS1235: Invalid regex

| Property | Value       |
| -------- | ----------- |
| Id       | RCS1235     |
| Category | Readability |
| Severity | Warning     |

## Example

### Code with Diagnostic

```csharp
Regex.Match("foobar", @"\pXXX");
```

## See Also

* [How to Suppress a Diagnostic](../HowToConfigureAnalyzers.md#how-to-suppress-a-diagnostic)


*\(Generated with [DotMarkdown](http://github.com/JosefPihrt/DotMarkdown)\)*